<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyQt Pro Translator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="settingsLayout">
      <item>
       <widget class="QComboBox" name="comboMode">
        <item>
         <property name="text">
          <string>Online (Google/DeepL)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Offline (Argos Neural)</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboSource"/>
      </item>
      <item>
       <widget class="QLabel" name="labelArrow">
        <property name="text">
         <string>→</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboTarget"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="textLayout">
      <item>
       <widget class="QTextEdit" name="TextInput">
        <property name="placeholderText">
         <string>Введите текст здесь...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="TextOutput">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Перевод появится здесь...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnTranslate">
      <property name="text">
       <string>ПЕРЕВЕСТИ</string>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="minimumHeight">
       <number>50</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
<resources/>
<connections/>
</ui>
